Userland/Libraries/LibSoftGPU/Clipper.cpp

```diff
@@ -91,6 +91,20 @@ FLATTEN static constexpr void clip_plane(Vector<GPU::Vertex>& input_list, Vector
     }
 }
 
+void Clipper::clip_points_against_frustum(Vector<GPU::Vertex>& vertices)
+{
+    Vector<GPU::Vertex> output_vertices;
```

Comment:
...and maybe consider starting with a heuristic for the size of this to avoid repeated reallocation (especially since this is called every frame)

Reply:
`Clipper` already had a `m_vertex_buffer` for exactly this purpose, I just forgot to use it. Changed!